aiida/cmdline/commands/cmd_verdi.py

```diff
@@ -38,22 +38,24 @@
 )
 
 
-class MostSimilarCommandGroup(click.Group):
-    """
-    Overloads the get_command to display a list of possible command
-    candidates if the command could not be found with an exact match.
+class VerdiCommandGroup(click.Group):
+    """Command group for verdi commands.
+
+    Overloads the get_command to display a list of possible command candidates if the command could not be found with an
+    exact match, and adds an option to control verbosity.
```

Comment:
Maybe good to add why we add this in this manner, due to the limitation in `click` that is by design.

Reply:
Will add!